Makefile

```diff
@@ -70,15 +74,14 @@ endif
 	@./tests/validate/whitespace.sh
 	@./tests/validate/govet.sh
 	@./tests/validate/git-validation.sh
-	@./tests/validate/gometalinter.sh . cmd/buildah
+	lint
```

Comment:
I think I'll probably need to make an equivalent gometalinter.sh for the new lint.  Will work on this more tomorrow.

Reply:
I guess a single invocation of `golangci-lint run` could be enough here.